libs/designsystem/toggle/src/toggle.component.scss

```
@@ -46,6 +46,17 @@ ion-toggle {
   --handle-background-checked: #{utils.get-color($handle-background)};
   --handle-transition: #{utils.get-transition-duration('quick')};
   --handle-box-shadow: #{utils.get-elevation(2)};
+
+  // Ion v8 changed height on toggles to 31px. To make toggles still follow our 8,16,24,32 pixel-size pattern, we force the toggle back to 32px
+  $toggle-size: utils.size('l');
+
+  --handle-height: calc(#{$toggle-size} - (2px * 2));
+  --handle-width: calc(#{$toggle-size} - (2px * 2));
```

Consider simplifying the syntax. Sass supports inline calculations, but the entire expression needs to be interpolated because of the css variable:
```suggestion
  --handle-height: #{$toggle-size - (2px * 2)};
  --handle-width: #{$toggle-size - (2px * 2)};
```